test/DAREditor/JSONDescrepency.cs

```diff
@@ -0,0 +1,25 @@
+﻿// // Copyright (c) Microsoft. All rights reserved.
+// // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Newtonsoft.Json;
+using OpenDebug;
+namespace DAREditor
+{
+    class JSONDescrepency
+    {
+        public static object inputJSon(string input, out Exception error)
```

Comment:
>inputJSon [](http://example.com/codeflow?start=29&length=9)

This should be capitalized as `InputJson`. Though I think a better name would probably be `TryDeserialize`.